src/features/people/employees/components/step1/step1Form.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import { Form } from 'antd';
+import PropTypes from 'prop-types';
+
+import {
+  getNickNameInput,
+  getFullNameInput,
+  getGenderSelector,
+  getMotherNameInput,
+  getFatherNameInput,
+  getNationalityInput,
+  getBirthDatePicker,
+  getMaritalStatusInput,
+  getSpouseNameInput,
+  getChildrenAmountInput,
+} from './step1Entries';
+
+import { getDecoratorManager } from './step1Decorators';
+import { LABELS } from './step1Constants';
+import {
+  FORM_ITEM_LAYOUT,
+  FORM_ITEM_LAYOUT_2,
+  HORIZONTAL_FORM_LAYOUT,
+} from '../stepFormHelper';
+
+import StepFormNavigator from '../stepFormNavigator';
+
+const FormItem = Form.Item;
+
+class NewEmployeeStep1Form extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleOnSubmit = this.handleOnSubmit.bind(this);
+  }
+
+  handleOnSubmit(event) {
+    event.preventDefault();
+    const { form } = this.props;
+
+    form.validateFields(error => {
+      if (!error) {
+        this.props.nextCallback();
+      }
+    });
+  }
+
+  render() {
+    const { getFieldDecorator: fieldDecorator } = this.props.form;
+    const decoratorManager = getDecoratorManager(fieldDecorator);
+
+    return (
+      <Form layout={HORIZONTAL_FORM_LAYOUT} onSubmit={this.handleOnSubmit}>
+        <FormItem {...FORM_ITEM_LAYOUT} label={LABELS.FULL_NAME} hasFeedback>
+          {decoratorManager.fullNameDecorator(getFullNameInput())}
+        </FormItem>
+
+        <FormItem {...FORM_ITEM_LAYOUT} label={LABELS.NICK_NAME} hasFeedback>
+          {decoratorManager.nickNameDecorator(getNickNameInput())}
+        </FormItem>
+
+        <FormItem {...FORM_ITEM_LAYOUT_2} label={LABELS.GENDER} hasFeedback>
+          {decoratorManager.genderDecorator(getGenderSelector())}
+        </FormItem>
+
+        <FormItem {...FORM_ITEM_LAYOUT_2} label={LABELS.BIRTH_DATE}>
+          {decoratorManager.birthDateDecorator(getBirthDatePicker())}
+        </FormItem>
+        <FormItem
+          {...FORM_ITEM_LAYOUT}
+          label={LABELS.FILIATION_FATHER}
+          hasFeedback
+        >
+          {decoratorManager.fatherNameDecorator(getFatherNameInput())}
+        </FormItem>
+
+        <FormItem
+          {...FORM_ITEM_LAYOUT}
+          label={LABELS.FILIATION_MOTHER}
+          hasFeedback
+        >
+          {decoratorManager.motherNameDecorator(getMotherNameInput())}
+        </FormItem>
+
+        <FormItem {...FORM_ITEM_LAYOUT} label={LABELS.NATIONALITY} hasFeedback>
+          {decoratorManager.nationalityDecorator(getNationalityInput())}
+        </FormItem>
+
+        <FormItem
+          {...FORM_ITEM_LAYOUT}
+          label={LABELS.MARITAL_STATUS}
+          hasFeedback
+        >
+          {decoratorManager.maritalStatusDecorator(getMaritalStatusInput())}
+        </FormItem>
+
+        <FormItem {...FORM_ITEM_LAYOUT} label={LABELS.SPOUSE_NAME} hasFeedback>
+          {getSpouseNameInput()}
+        </FormItem>
+
+        <FormItem
```

Comment:
the placeholder doesn't hide when the user starts typing
![image](https://user-images.githubusercontent.com/9323529/34582048-60e02f3e-f171-11e7-862a-c71f963da90a.png)